integration/apps/hanami/.envrc.sample

```
@@ -1,2 +1,3 @@
 export DD_API_KEY=<Your Datadog API key here>
 export DD_HOSTNAME=$(hostname)
+export RUBY_VERSION=<Ruby version you are testing>
```

I suggested provided a benign default value for the file. 

```suggestion
export RUBY_VERSION=2.7
```

---

👍